plugins/destination/gremlin/client/spec_test.go

```diff
@@ -0,0 +1,132 @@
+package client
+
+import (
+	"testing"
+
+	"github.com/cloudquery/codegen/jsonschema"
+)
+
+func TestJSONSchema(t *testing.T) {
+	jsonschema.TestJSONSchema(t, JSONSchema, []jsonschema.TestCase{
+		{
+			Name: "empty spec",
+			Spec: `{}`,
+			Err:  true,
+		},
+		{
+			Name: "spec with endpoint",
+			Spec: `{"endpoint": "ws://localhost:8182"}`,
+		},
+		{
+			Name: "spec with wss endpoint, without port",
+			Spec: `{"endpoint": "wss://localhost"}`,
+		},
+		{
+			Name: "spec with bad endpoint 1",
+			Spec: `{"endpoint": "http://localhost:8182"}`,
```

Comment:
I'm not sure we need to validate port ourselves, as the Validate logic didn't do it in the first place

Reply:
removed